luup files/L_RGBController1.lua

```diff
@@ -1037,6 +1037,49 @@ function doColorTransition(lul_device)
 	end
 end
 
+-- MySensor RGB(W)
+RGBDeviceTypes["MYS-RGBW"] = {
+	getParameters = function (lul_device)
+		return {
+			name = "MySensors RGBW",
+			settings = {
+				{ variable = "ArduinoId", name = "Arduino plugin Id", type = "string" },
+				{ variable = "RadioId",   name = "RGB Node altid",    type = "string" }
+			}
+		}
+	end,
+
+	getColorChannelNames = function (lul_device)
+		return {"red", "green", "blue", "warmWhite", "coolWhite"}
```

Comment:
Hello Bekkie,
I am not used to the pull requests yet.
It seems that you have not commited all the code (it lacks "setStatus" and "setColor" functions).

Besides, I'm not sure to understand your installation process. You don't have to delete the RGB color wheel created by Arduino plugin (which I don't know). I think the RGB controller can work with these devices.
And to create another RGB Controller, you just have to go in "Apps" panel, you will find a button "create another".